app/src/main/java/com/example/topmovies/models/domain/MovieDetails.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.topmovies.models.domain
+
+data class MovieDetails(
+    val id: String,
+    val title: String,
+    val year: String,
+    val plot: String,
+    val imageUrl: String,
+    val releaseDate: String,
+    val runtimeStr: String,
+    val genres: String,
+    val imDbRating: String,
+    val errorMessage: String? = null
+) {
+    companion object {
```

Comment:
If you need in some cases empty model, you can set values by default, like that : 
val id: String = "",
val title: String = "",
.......